ports/libsodium/vcpkg.json

```
@@ -1,7 +1,7 @@
 {
   "name": "libsodium",
   "version-string": "1.0.18",
```

```suggestion
  "version": "1.0.18",
```
This field should be `version `instead of `version-string`. Please see https://github.com/microsoft/vcpkg/blob/master/docs/users/versioning.md.

---

Fixed. 👍